inst/rmarkdown/templates/jds/skeleton/skeleton.Rmd

```diff
@@ -0,0 +1,271 @@
+---
+title: |
+  My Long, Long, JDS Paper Title That May
+  Take not One but Two or not Two but Three or not Three but Four or
+  Not Four but Five Lines
+shorttitle: Short JDS Paper Title
+author:
+- name: Adam Ahen
+  affiliation: "1,2"  # Must be quoted when including comma.
+  email: "foo1@bar.com"  # Only corresponding authors require email.
+- name: Brett Berger
+  affiliation: 2
+  email: "foo2@bar.com"  # Only corresponding authors require email.
+- name: Carl Camp
+  affiliation: 3
+- name: David Dodge
+  affiliation: 3
+
+affiliation:
+- id: 1
+  institution: "Department of Statistics, University of Achievement, Street Address, Country"
+- id: 2
+  institution: "Institution B"
+- id: 3
+  institution: "Institution C"
+
+abstract: |
+  This article presents a classification of disease severity for
+  patients with cystic fibrosis (CF). CF is a genetic disease that dramatically
+  decreases life expectancy and quality.  The disease is characterized by
+  polymicrobial infections which lead to lung remodeling and airway mucus
+  plugging. In order to quantify disease severity of CF patients and compute a
+  continuous severity index measure, quantile regression, rankscores, and
+  corresponding normalized ranks are calculated for CF patients. These ranks
+  make use of the conditional distribution of lung function across a range of
+  quantiles in order to quantify disease severity of CF.
+keywords: 
+  - Conditional distribution
+  - Cystic fibrosis
+  - Quantiles
+
+# the following edits should be done by Journal typesetters
+page: 1  # set the first page number
+month: July
+year: 2020
+volume: xx
+issue: xx
+doi: "xx.xxxx/xxxxxxxxx"
+received: "April, 2020"  # optional: comment it out if no received date
+accepted: "May, 2020"  # optional: comment it out if no accepted date
+
+bibliography: JDSbib.bib
+output:
+  rticles::jds_article: default
+  bookdown::pdf_book:
+    base_format: rticles::jds_article # for using bookdown features like \@ref()
+---
+
+```{r setup, include = FALSE, echo = FALSE, purl = FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+
+# deactivate addition of preamble when this format is used with bookdown
+options(bookdown.theorem.preamble = FALSE)
```

Comment:
I believe this should be done always for this format right ? 

I think we should add this option to be set in the format function directly so that user does not need to care about that. 

What do you think ?